DEVELOPMENT.md

```diff
@@ -195,7 +195,7 @@ This step includes building Knative Serving, creating and pushing developer
 images and deploying them to your Kubernetes cluster. If you're developing
 locally (for example, using
 [Docker-on-Mac](https://knative.dev/docs/install/knative-with-docker-for-mac/)),
-set `KO_DOCKER_REPO=ko.local` (or `KO_DOCKER_REPO=kind.local` respectively to
+set `KO_DOCKER_REPO=ko.local` (or `KO_DOCKER_REPO=kind.local`) respectively to
```

Comment:
```suggestion
set `KO_DOCKER_REPO=ko.local` (or `KO_DOCKER_REPO=kind.local` respectively) to
```

"respectively" belongs to the alternative and thus needs to be inside the parentheses.